lib/pg_query/parse.rb

```diff
@@ -48,27 +49,48 @@ def ddl_tables
     tables_with_types.select { |t| t[:type] == :ddl }.map { |t| t[:table] }
   end
 
+  # Returns function names, ignoring their argument types. This may be insufficient
+  # if you need to disambiguate two functions with the same name but different argument
+  # types.
+  def functions
+    functions_with_types.map { |f| f[:function] }
+  end
+
+  def ddl_functions
+    functions_with_types.select { |f| f[:type] == :ddl }.map { |f| f[:function] }
+  end
+
+  def call_functions
```

Comment:
I tried to name these methods following the pattern of `select_tables`, `ddl_tables`...